src/views/product-downloads-view/helpers.ts

```diff
@@ -359,14 +360,42 @@ export const generateFeaturedTutorialsCards = async (
 		const { productSlug, collectionSlug, tutorialSlug } =
 			splitTutorialSlug(slug)
 		const tutorialData = inlineTutorials[`${productSlug}/${tutorialSlug}`]
+
+		/**
+		 * We have full `Tutorial` data, but we need `TutorialLite` specifically,
+		 * with a `defaultContext` property, to satisfy `formatTutorialCard`.
+		 *
+		 * Note: We could potentially change `TutorialLite` to maintain the
+		 * `collectionCtx` property from `Tutorial`? This would remove the
+		 * need to adapt data in this way, which we do in a few places
+		 * for `formatTutorialCard`. Asana task:
+		 * https://app.asana.com/0/1202097197789424/1202964504180140/f
+		 */
 		const defaultContext = tutorialData.collectionCtx.default
 		const tutorialLiteCompat = { ...tutorialData, defaultContext }
 
-		const formattedTutorialCard = formatTutorialCard(tutorialLiteCompat)
-		if (collectionSlug) {
-			formattedTutorialCard.url = `/${productSlug}/tutorials/${collectionSlug}/${tutorialSlug}`
+		/**
+		 * We have a collection context to use other than the default.
+		 */
+		let collectionContext = tutorialData.collectionCtx.default
```

Comment:
Updated the helper here to work with the revised `formatTutorialCard`.

The refactor here didn't feel the cleanest, so tried to be thorough with comments, and have also added this task (also linked to in code comments): 🎟 [[types] consider refactoring TutorialLite type](https://app.asana.com/0/1202097197789424/1202964504180140/f).